.vscode/settings.json

```diff
@@ -1,12 +1,11 @@
 {
```

Comment:
I hadn't realized that we had committed the vscode settings. Maybe this should be ignored.

Reply:
There's literally a force include line in the gitignore.

I thought about this a bit and I think we should def include the `.vscode` stuff -- I think we should recommend VSCode in our getting started. Hopefully new people can just clone the repo, add the recommended extensions and be ready to go. It's also helpful to make sure the plugin settings match the lint rule settings.

I also have started putting launch scripts in `launch.json`. It's super easy to get the dev server up in debug mode.